Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -2501,6 +2534,22 @@ void webkitWebViewRunAsModal(WebKitWebView* webView)
 #endif
 }
 
+bool webkitWebViewHasMainResource(WebKitWebView* webView)
```

Comment:
This is actually an exception to the usual rule that we should use pass by reference to indicate the parameter is not nullable. For better or for worse, we have an unwritten rule that GObjects always use pass by pointer. We should probably change that, because it's not a good rule, but not today. Anyway, I'm calling this out because I asked you to change pass by pointer to pass by reference up above, and you'd probably be tempted to change this here as well, but please leave this as-is due to the GObject rule.